### PR TITLE
Latest Topics & profile edit interest Apollo Hook Switch

### PR DESCRIPTION
**Pull request by themonster2015**
## Description

* Refactor Latest Topics widget and switch to apollo hooks
* Switch to Apollo hook on profile edit interest page

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: ui/widgets/trending/src/components/cards/latest-topics.tsx [hunk 1]
```diff
@@ -29,27 +31,203 @@ export type LatestTopicsProps = {
...
+                  labelType: sdk.services.gql.labelTypes.INTEREST,
+                })),
+              },
+            },
+          },
+          optimisticResponse: {
+            updateAkashaProfileInterests: {
+              clientMutationId: null,
+              document: {
+                did: { id: 'did:pkh:eip155:5:0xe92bbe3e927f73106e57ed43fd3f2acf51035128' },
```
Comment (kenshyx) [inline ui/widgets/trending/src/components/cards/latest-topics.tsx]: why this specific DID?

Comment (kenshyx) [reply ui/widgets/trending/src/components/cards/latest-topics.tsx]: you could use something like 0x address `did:pkh:eip155:1:0x0000000000000000000000000000000000000000`

Comment (kenshyx) [reply ui/widgets/trending/src/components/cards/latest-topics.tsx]: or even better, fetch the DID from the currently signed in user `getCurrentUser`

Comment (kenshyx) [reply ui/widgets/trending/src/components/cards/latest-topics.tsx]: this should not be hardcoded like here

Comment (themonster2015) [reply ui/widgets/trending/src/components/cards/latest-topics.tsx]: thanks for spotting this.

**Review by kenshyx — APPROVED** [review]

**Pull request merged**